completed sidebar

Preet2099x opened the pull request.

RohitRaj766 reviewed (commented):
in sidebar.scss what is the purpose of code writen from line 3 to 31 ? 
in sidear.jsx line 31 there is spelling mistake, its Qezee.

Preet2099x commented:
> in sidebar.scss what is the purpose of code writen from line 3 to 31 ? in sidear.jsx line 31 there is spelling mistake, its Qezee.

It was left there by mistake. I removed it in the 2nd commit (Upated Sidebar.scss). 
Fixed the spelling mistake in the 3rd commit. (The spelling mistake is present in Header.jsx as well)
